noir-projects/noir-contracts/contracts/lending_contract/src/position.nr

```diff
@@ -1,13 +1,14 @@
 use dep::aztec::protocol_types::traits::{Deserialize, Serialize};
 
 pub struct Position {
-    collateral: Field,
-    static_debt: Field,
-    debt: Field,
+    collateral: U128,
+    static_debt: U128,
+    debt: U128,
 }
 
 global POSITION_SERIALIZED_LEN: u32 = 3;
 
+// TODO(benesjan): either don't call to_field below or make this Packable trait instead.
```

Comment:
Will address in a PR in which I implement the Packable trait: https://github.com/AztecProtocol/aztec-packages/pull/11136